healthtip/main.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"database/sql"
 	"fmt"
+	"healthtip-demo"
```

Comment:
I like the reorganization but the file organization actually should be in reverse (in idiomatic Go). 

`main.go` should be in the root directory then you could create a package directory called `healthtip` or `healthtip-demo` and put the packages files that need to be called like `db.go` and `mailgun.go` etc. in there

you could put the `web.go` file and associated handlers in that package or separate it out into its own package, but that would probably necessitate rethinking where mailgun.go goes as well, maybe not

at a minimum to keep this reorganization we should put `main.go` in the root directory and the rest of the files in a subdirectory and have that be the `healthtip` package